superset/assets/src/SqlLab/App.jsx

```diff
@@ -20,14 +20,27 @@ setupApp();
 const appContainer = document.getElementById('app');
 const bootstrapData = JSON.parse(appContainer.getAttribute('data-bootstrap'));
 initFeatureFlags(bootstrapData.common.feature_flags);
-const state = getInitialState(bootstrapData);
+const initialState = getInitialState(bootstrapData);
+const sqlLabPersistStateConfig = {
+  paths: ['sqlLab'],
+  config: {
+    slicer: paths => (state) => {
+      const subset = {};
+      paths.forEach((path) => {
+        delete state[path].common; // eslint-disable-line no-param-reassign
```

Comment:
can you add a comment explaining what is going on here so people that don't have context can understand it?

also, what's the plan for removing this once it becomes dead code? can we safely delete it in a month? 2 months?

Reply:
this is a good question..we don't have many options on browser's localStorage if user didn't use sql lab. but if user didn't open sql lab for 2 months he probably didn't critically depend on localStorage.

this PR will help @jeffreythewang's feature: https://github.com/apache/incubator-superset/pull/4941. So I am thinking after a few new features in Sql Lab (a month long is also good), we can remove this cleanup line.